modules/distributor/distributor.go

```diff
@@ -272,7 +272,7 @@ func (d *Distributor) checkForRateLimits(tracesSize, spanCount int, userID strin
 	if !d.ingestionRateLimiter.AllowN(now, userID, tracesSize) {
 		overrides.RecordDiscardedSpans(spanCount, reasonRateLimited, userID)
 		return status.Errorf(codes.ResourceExhausted,
-			"%s ingestion rate limit (%d bytes) exceeded while adding %d bytes",
+			"%s ingestion rate limit (%d bytes) exceeded while adding %d bytes for user %s",
```

Comment:
Nice, since you're adding a new variable `%s`, then you'll also need to add the `userID` variable to the end, before the find `)`.  Basically the number of variables in the string need to match the number of arguments you pass.

Reply:
Perfetc.